test/test_helper.rb

```diff
@@ -8,7 +8,9 @@
 require 'minitest/autorun'
 require 'minitest/unit'
 require 'minitest/pride' if ENV['RAINBOW_COLORED_TESTS'] == "1" && $stdout.tty?
+require 'minitest/stub_const'
 require 'mocha/setup'
+require 'byebug'
```

Comment:
I don't want to run my tests with a debugger on by default.

Reply:
✅ leftover sry